frontend/src/App.js

```diff
@@ -92,12 +60,12 @@ function App() {
             <Route exact path={ROUTES.NEW_PROJECT}>
               <NewProject />
             </Route>
-            <ProposalProvider>
-              <Route exact path={ROUTES.MY_PROJECTS}>
-                <MyProjects />
-              </Route>
-            </ProposalProvider>
             <EOIProvider>
+              <ProposalProvider>
```

Comment:
No longer need proposal provider, since I have moved the proposal from ProposalProvider to ProjectProvider.